rego/testdata/Makefile

```diff
@@ -0,0 +1,5 @@
+# NOTE(sr): this is the last image emitting wasm modules with ABI 1.1
+IMAGE := openpolicyagent/opa:0.30.1
+
+bundle.tar.gz: t.rego
```

Comment:
I didn't see this hooked up anywhere. Do we want to keep this directory? Same for `tmp.wasm` above.

Reply:
It's not hooked up, but the generated file is used here: https://github.com/open-policy-agent/opa/pull/3627/files#diff-98755a90ab5e33f048eb9d54795984851c84b9697b16d9b18e5083753eee10ebR278

I thought it better to not commit generated data without the instructions on how to generate it.

Reply:
☝️ @tsandall just realised I've merged this without having resolved that thing. I'll do a cleanup PR if you think differently about generated files there! 😃